.browserslistrc

```diff
@@ -11,3 +11,7 @@ last 2 Edge major versions
 last 2 Safari major versions
 last 2 iOS major versions
 Firefox ESR
+not IE 9-10
+not IE 11
+not ios_saf 15.2-15.3
+not safari 15.2-15.3
```

Comment:
This has nothing to do with issue #11191 - it's probably better to separate these changes out into their own fix.